odoo/addons/base/models/ir_ui_view.py

```diff
@@ -556,24 +571,34 @@ def get_inheriting_views_arch(self, view_id, model):
                 if not view.groups_id or (view.groups_id & user_groups)]
 
     @api.model
-    def raise_view_error(self, message, view_id):
-        view = self.browse(view_id)
-        not_avail = _('n/a')
-        message = (
-            "%(msg)s\n\n" +
-            _("Error context:\nView `%(view_name)s`") +
-            "\n[view_id: %(viewid)s, xml_id: %(xmlid)s, "
-            "model: %(model)s, parent_id: %(parent)s]"
-        ) % {
-            'view_name': view.name or not_avail,
-            'viewid': view_id or not_avail,
-            'xmlid': view.xml_id or not_avail,
-            'model': view.model or not_avail,
-            'parent': view.inherit_id.id or not_avail,
-            'msg': message,
-        }
-        _logger.info(message)
-        raise ValueError(message)
+    def raise_view_error(self, message, view, recurse=False):
+        log_view_error(message, view)
+        if not view and recurse:
```

Comment:
I think that `recurse` could be derived from the context:
```suggestion
        if not view and not view.env.context.get('inherit_branding'):
```